docs/conf.py

```diff
@@ -70,12 +70,31 @@
     ("py:class", "sqlalchemy.orm.decl_api.DeclarativeMeta"),
     ("py:class", "sqlalchemy.sql.sqltypes.TupleType"),
     ("py:class", "sqlalchemy.dialects.postgresql.named_types.ENUM"),
+    ("py:class", "_orm.Mapper"),
+    ("py:class", "_orm.registry"),
+    ("py:class", "_schema.MetaData"),
+    ("py:class", "_schema.Table"),
+    ("py:class", "_RegistryType"),
+    ("py:class", "abc.Collection"),
+    ("py:class", "TypeEngine"),
+    ("py:class", "ExternalType"),
+    ("py:class", "UserDefinedType"),
+    ("py:class", "_types.TypeDecorator"),
+    ("py:meth", "_types.TypeDecorator.process_bind_param"),
+    ("py:meth", "_types.TypeDecorator.process_result_value"),
+    ("py:meth", "type_engine"),
     # type vars and aliases / intentionally undocumented
     ("py:class", "RouteHandlerType"),
     ("py:obj", "litestar.security.base.AuthType"),
     ("py:class", "ControllerRouterHandler"),
     ("py:class", "PathParameterDefinition"),
     ("py:class", "BaseSessionBackendT"),
+    ("py:class", "litestar.contrib.repository.abc.CollectionT"),
+    ("py:obj", "ModelT"),
+    ("py:obj", "litestar.contrib.sqlalchemy.repository.ModelT"),
+    ("py:class", "litestar.contrib.sqlalchemy.repository.ModelT"),
+    ("py:class", "ModelT"),
```

Comment:
Instead of having both `py:class` and `py:obj` for `ModelT`, you can just use a regex in `nitpick_ignore_regex`: `("py:.*":, "ModelT")`

Reply:
Yep - i'll update this morning.